### PR TITLE
MWPW-113002 Adding up patterns to charts

### PR DESCRIPTION
**Pull request by Brandon32**
* Add chart section class
* Add 1-up, 2-up and 3-up

Resolves: [MWPW-113002 ](https://jira.corp.adobe.com/browse/MWPW-113002)

**Test URLs:**
- Before: https://data-viz--milo--adobecom.hlx.page/drafts/bmarshal/chart
- After: https://chart-up--milo--adobecom.hlx.page/drafts/bmarshal/chart

### CODE REVIEW

File: libs/blocks/chart/chart.css [hunk 1]
```diff
@@ -3,6 +3,34 @@
   to { transform: scale(1); }
 }
 
+.section.chart-section {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: flex-start;
+    margin-left: auto!important;
```
Comment (chrischrischris) [inline libs/blocks/chart/chart.css:11]: Why the `!important`?  Overriding echarts css?

Comment (Brandon32) [reply libs/blocks/chart/chart.css]: Actually these aren't needed, copied them over on accident. Thanks.

File: libs/blocks/chart/chart.css [hunk 2]
```diff
@@ -3,6 +3,32 @@
   to { transform: scale(1); }
 }
 
+.section.chart-section {
+    display: flex;
```
Comment (meganthecoder) [inline libs/blocks/chart/chart.css:7]: I believe flexbox needs prefixes

File: libs/blocks/chart/chart.js [hunk 3]
```diff
@@ -189,6 +190,7 @@ const handleIntersect = (chart, chartOptions) => (entries, observer) => {
 
   entries.forEach((entry) => {
     if (entry.isIntersecting) {
+      chart?.resize();
```
Comment (meganthecoder) [inline libs/blocks/chart/chart.js:193]: Why is the chart resizing before the options are even set? There's no chart to resize

Comment (Brandon32) [reply libs/blocks/chart/chart.js:193]: The chart is created before the observer, flexbox is changing the width so I have to resize. 
If I resize after setting the options then the animation gets canceled.

**Comment by Brandon32** [comment]
Going to open a new PR, looks like helix isn't loading the chart theme so the charts don't load.
https://chartup--milo--adobecom.hlx.page/libs/blocks/chart/chartLightTheme.js

File: libs/blocks/chart/chart.js (hunk 3 above)
Comment (meganthecoder) [reply libs/blocks/chart/chart.js:193]: Ah, in that case, it seems like we shouldn't be initializing the chart ahead of time. Instead do the init in the observer handler

**Pull request closed without merging**